index.html

```diff
@@ -144,8 +144,7 @@ <h2>
             </li>
           </ol>
         </li>
-        <li>Let <var>max length</var> be an implementation-dependent maximum
-        length of <var>pattern</var>.
+        <li>Let <var>max length</var> have the value 10.
           <div class='note'>
```

Comment:
This note needs to be updated as well, since right now it suggests that implementations might ignore or work around the maximum pattern length. Since the normative text below says to truncate, I think we can just remove this note.

Reply:
Good catch, addressed by https://github.com/w3c/vibration/pull/46/commits/a3af007daf49001bb924a6d345e5dbc2a0c6d96f